PLANNING_GROUP: manipulator
frame_id: iiwa_link_0
move_step: 0.15
lift_step: 0.10
planning_time: 2
roll_step: 0.5
pitch_step: 0.5
yaw_step: 0.4
